src/Host.cpp

```diff
@@ -2463,6 +2465,17 @@ const QString& Host::getDiscordApplicationID()
     return mDiscordApplicationID;
 }
 
+void Host::setDiscordInviteURL(const QString& s)
+{
+    mDiscordInviteURL = s;
+    writeProfileData(QStringLiteral("discordInviteURL"), s);
+}
+
+const QString& Host::getDiscordInviteURL()
```

Comment:
At least consider this suggestion (and a matching one for the declaration in the `Host.h` file):
```suggestion
const QString& Host::getDiscordInviteURL() const
```
however I think it is even better to move this simple *getter* to the header file entirely - there are plenty of others of the same generic format already there - and they too should {but apart from a stopwatch one,  **don't**, :man_shrugging:) have the `const` *after* the method name {to flag that they are not capable of modifying the class's own members}!